content/docs/user-guide/overview.md

```diff
@@ -108,8 +108,8 @@ bringing best practices from software engineering into the data science field.
 \* **copy-on-write links or "reflinks"** are a relatively new way to link files
 in UNIX-style file systems. Unlike hardlinks or symlinks, they support
 transparent [copy on write](https://en.wikipedia.org/wiki/Copy-on-write). This
```

Comment:
```suggestion
\* ([copy-on-write](https://en.wikipedia.org/wiki/Copy-on-write)) links or
**reflinks** are a type of file linking available in UNIX-style file systems.
Unlike hardlinks or symlinks, these file entries turn into a plain copy when
```

Reply:
OK well I'm going to commit my suggestion since we need to fix the formatting anyway.